app/src/main/res/layout/fragment_by_country.xml

```diff
@@ -15,9 +15,9 @@
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="Country Cases"
+            android:text="@string/country_fragment_country_cases"
```

Comment:
strings.xml is the best place for string literals.  You can have a different strings.xml for each language you support, and the Android operating system will automatically choose the correct file, based on the user's preferred language.